### ISSUE TITLE
[zookeeper] 3.3.0+ can use a better stats command

### ISSUE THREAD

**Issue by miketheman** [DataDog/dd-agent#574]
Old version of `stat` command returned 
Version 3.4.x now includes a better command, `mntr` that provides something like:

```
zk_version  3.4.0
zk_avg_latency  0
zk_max_latency  0
zk_min_latency  0
zk_packets_received 70
zk_packets_sent 69
zk_outstanding_requests 0
zk_server_state leader
zk_znode_count   4
zk_watch_count  0
zk_ephemerals_count 0
zk_approximate_data_size    27
zk_followers    4                   - only exposed by the Leader
zk_synced_followers 4               - only exposed by the Leader
zk_pending_syncs    0               - only exposed by the Leader
zk_open_file_descriptor_count 23    - only available on Unix platforms
zk_max_file_descriptor_count 1024   - only available on Unix platforms
```

Would make parsing a lot cleaner.
Ref: http://zookeeper.apache.org/doc/r3.4.5/zookeeperAdmin.html#sc_zkCommands

**Comment by rthille** [comment]
Any reason why the JMX interface isn't used?

**Comment by miketheman** [comment]
Resolved via #2156
